src/components/DeleteRequestModal.svelte

```diff
@@ -0,0 +1,28 @@
+<script>
+    import { Dialog } from '@silintl/ui-components'
```

Comment:
Not that you'll want to fix it in this release PR necessarily, but there's some odd indentation in this file.

Reply:
thanks for bringing that to my attention, since I have switched computers so many times I had not set indentation to 2 spaces yet.